extension/src/telemetry/index.test.ts

```diff
@@ -1,4 +1,5 @@
 import { extensions } from 'vscode'
+// eslint-disable-next-line import/default
```

Comment:
![image](https://user-images.githubusercontent.com/37993418/176592214-d7186f7e-5cde-4877-9b7d-87bda059ce65.png)

Reply:
Verified that it is still working with the built VSIX:

![image](https://user-images.githubusercontent.com/37993418/176593184-3cb28824-cac1-4283-be45-ff423ef18950.png)